src/hooks/useRegistryInformation.jsx

```diff
@@ -3,20 +3,17 @@ import { Registry } from 'cozy-client'
 
 const useRegistryInformation = (client, slug) => {
   const [appData, setAppData] = useState(null)
-  const registry = new Registry({
-    client
-  })
 
-  useEffect(
-    () => {
-      const fetchData = async () => {
-        const app = await registry.fetchApp(slug)
-        setAppData(app)
-      }
-      fetchData()
-    },
-    [slug]
-  )
+  useEffect(() => {
+    const fetchData = async () => {
+      const registry = new Registry({
```

Comment:
Nit: This might be a sign that instanciating the `Registry` during the components lifecycle may not be a good idea, it might be more react-friendly to provide an instance as a props somehow.

Reply:
I think we could instantiate the Registry directly in cozy-client (we should check how much the registry is adding to the bundle size). Or as a plugin. But registry should be fairly light.